pty/src/serial.rs

```diff
@@ -116,6 +116,17 @@ impl SlavePty for Slave {
             port: Arc::clone(&self.port),
         }))
     }
+        /// Spawns a command in the security context of the primary token
+    #[cfg(windows)]
+    fn spawn_command_as_user(&self, cmd: CommandBuilder, h_token: &winapi::um::winnt::HANDLE) -> anyhow::Result<Box<dyn Child + Send + Sync>> {
+        ensure!(
```

Comment:
FWIW, I think this should unconditionally error out with an explanation that it is unsupported for serial ports